### PR TITLE
Add test run in Docker for Azure.Identity tests

### PR DESCRIPTION
**Pull request by AlexanderSher**

### CODE REVIEW

File: sdk/identity/tests.yml [hunk 1]
```diff
@@ -3,6 +3,21 @@ trigger: none
 extends:
   template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
   parameters:
+    AdditionalPlatforms:
+      displayName: 'Test on Linux with keyring'
+      vmImage: 'ubuntu-18.04'
```
Comment (weshaggard) [inline sdk/identity/tests.yml:8]:
```suggestion
      OSVmImage: 'ubuntu-18.04'
```

**Comment by AlexanderSher** [comment]
Tests have passed

File: sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialLiveTests.cs [hunk 2]
```diff
@@ -82,7 +82,7 @@ public async Task AuthenticateWithVscCredential_EmptySettingsFile()
         }
 
         [Test]
-        [RunOnlyOnPlatforms(Windows = true, OSX = true)] // Comment this attribute to run this tests on Linux with Libsecret enabled
+        [RunOnlyOnPlatforms(Windows = true, OSX = true, ContainerNames = new[] { "ubuntu_netcore2_keyring" })]
```
Comment (AlexanderSher) [inline sdk/identity/Azure.Identity/tests/VisualStudioCodeCredentialLiveTests.cs:85]: @schaabs we finally have tests that run on linux with enabled keyring. Maybe other identity teams would need that too.

**Review by weshaggard — APPROVED** [review]
Looks good from EngSys prospective.

**Pull request merged**